docs/usage.md

```diff
@@ -74,6 +78,43 @@ RUN mkdir -p /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 COPY 50-weekly.conf /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
 ```
 
+## Air-gapped and dissconnected updates
+
+For environments without a direct connection to a centralized container
+registry, we encourage mirroring a registry in DMZ if possible or manually
```

Comment:
I am not entirely sure how to interpret DMZ.  Should we point readers to /etc/containers/registries.conf [1] instead?

[1] https://www.redhat.com/sysadmin/manage-container-registries